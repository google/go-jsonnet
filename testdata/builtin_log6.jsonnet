std.log(0.0000000000000001)
