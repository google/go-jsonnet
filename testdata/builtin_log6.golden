-36.841361487904734
